inc/modules/export/epub/class-epub.php

```diff
@@ -775,8 +788,7 @@ protected function tidy( $html ) {
 	 *
 	 * @return string|array
 	 */
-	protected function fetchAndSaveUniqueMedia( $url, $fullpath ) {
-
+	protected function fetchAndSaveUniqueMedia( string $url, string $fullpath ) {
```

Comment:
@fdalcin in order to follow the proposed changes. Should we specify the type declaration for this method as well?